README-install.md

```diff
@@ -72,6 +72,51 @@ file like `/etc/sudoers.d/999-cloud-init-requiretty` with the following contents
     # cat /etc/sudoers.d/999-cloud-init-requiretty
     Defaults !requiretty
 
+Persistent Storage
+------------------
+
+Elasticsearch, Fluentd, and mux all require persistent storage - Elasticsearch
+for the database, and Fluentd and mux for the file buffering.  Fluentd and mux
+will use `/var/lib/fluentd` and `/var/log/fluentd`, respectively, by default.
+The disk space usage for Fluentd and mux will depend on how fast mux and
+Elasticsearch can process and ingest the logs.  Fluentd uses the `vars.yaml`
+(see below) parameter `openshift_logging_fluentd_file_buffer_limit` (default
+`1Gi` for 1 Gigabyte) to control the size of the file buffer, and mux uses
+`openshift_logging_mux_file_buffer_limit` (default `2Gi` for 2 Gigabytes).
```

Comment:
We may want to add the buffer_flie_limit is per output buffer.  If ops-cluster is enabled, the total limit would be doubled (e.g., 1Gi * 2 for fluentd; 2Gi * 2 for mux).